lib/core/src/sdk.rs

```diff
@@ -802,21 +802,31 @@ impl LiquidSdk {
         Ok(lbtc_pair)
     }
 
-    async fn get_broadcast_fee_estimation(&self, amount_sat: u64) -> Result<u64> {
+    /// Estimate the onchain fee for sending the given amount to the given destination address
+    async fn estimate_onchain_tx_fee(&self, amount_sat: u64, address: &str) -> Result<u64> {
+        Ok(self
+            .build_tx(None, address, amount_sat)
+            .await?
+            .all_fees()
+            .values()
+            .sum())
+    }
+
+    async fn estimate_lockup_tx_fee(&self, amount_sat: u64) -> Result<u64> {
+        self.estimate_onchain_tx_fee_p2tr(amount_sat).await
+    }
+
+    /// Estimate the onchain fee for sending the given amount to a taproot address
+    async fn estimate_onchain_tx_fee_p2tr(&self, amount_sat: u64) -> Result<u64> {
```

Comment:
Not sure we need this function, it is only use inside `estimate_lockup_tx_fee` perhaps inline it there?

Reply:
Done: 6e705dc5ec54dbcdfca239729a5559d247fe7936